cmd/cli/serve/serve.go

```diff
@@ -427,10 +432,10 @@ func serve(cmd *cobra.Command, OS *ServeOptions) error {
 		)
 
 		shellVariablesString := fmt.Sprintf(`
-export BACALHAU_IPFS_SWARM_ADDRESSES=%s
-export BACALHAU_API_HOST=%s
-export BACALHAU_API_PORT=%d
-export BACALHAU_PEER_CONNECT=%s`, ipfsSwarmAddress, OS.HostAddress, apiPort, peerAddress)
+			export BACALHAU_IPFS_SWARM_ADDRESSES=%s
+			export BACALHAU_API_HOST=%s
+			export BACALHAU_API_PORT=%d
+			export BACALHAU_PEER_CONNECT=%s`, ipfsSwarmAddress, OS.HostAddress, handler.GetAPIPort(ctx), peerAddress)
```

Comment:
this will add empty spaces to wherever we are printing these variables

Reply:
addressed.